engine/src/test/java/org/hibernate/validator/test/internal/engine/valuehandling/UnwrapValidatedValueTest.java

```diff
@@ -118,87 +113,6 @@ public void shouldRaiseExceptionIfNoMatchingUnwrapperIsFound() {
 		validator.validate( new Order() );
 	}
 
-	@Test
```

Comment:
`.unwrapValidatedValue()` is gone and everything is managed in the constraint definitions.